src/steamship/data/plugin.py

```diff
@@ -119,7 +119,7 @@ class PluginAdapterType(str, Enum):
 
 class PluginTargetType(str, Enum):
     file = "file"
```

Comment:
Enum's are typically all caps. 
```suggestion
    FILE = "file"
```

The effects of this change should be limited to the python-client, in case it's not let's leave it.